crates/deltalake-sql/src/parser.rs

```diff
@@ -167,50 +163,6 @@ impl<'a> DeltaParser<'a> {
         }
     }
 
-    /// Parse a SQL `DESCRIBE` statement
-    pub fn parse_describe(&mut self) -> Result<Statement, ParserError> {
-        match self.parser.peek_token().token {
-            Token::Word(w) => match w.keyword {
-                Keyword::DETAIL => {
-                    self.parser.next_token();
-                    let table = self.parser.parse_object_name()?;
-                    Ok(Statement::Describe(DescribeStatement {
-                        table,
-                        operation: DescribeOperation::Detail,
-                    }))
-                }
-                Keyword::HISTORY => {
-                    self.parser.next_token();
-                    let table = self.parser.parse_object_name()?;
-                    Ok(Statement::Describe(DescribeStatement {
-                        table,
-                        operation: DescribeOperation::History,
-                    }))
-                }
-                Keyword::FILES => {
-                    self.parser.next_token();
-                    let table = self.parser.parse_object_name()?;
-                    Ok(Statement::Describe(DescribeStatement {
-                        table,
-                        operation: DescribeOperation::Files,
-                    }))
-                }
-                _ => {
-                    let table = self.parser.parse_object_name()?;
-                    Ok(Statement::Datafusion(DFStatement::DescribeTableStmt(
-                        DescribeTableStmt { table_name: table },
-                    )))
-                }
-            },
-            _ => {
-                let table_name = self.parser.parse_object_name()?;
-                Ok(Statement::Datafusion(DFStatement::DescribeTableStmt(
-                    DescribeTableStmt { table_name },
-                )))
-            }
-        }
-    }
```

Comment:
this is some code to parse `DESCRIBE` statements specific to delta, so only the default branches (`_`) should be updated to fall back to the new behaviour in datafusion.

Reply:
actually. disregard that @philippemnoel. while the above is true, the sql crate is in such a raw state that it does not hurt much to remove this code for now, and bring it back properly once it actually gets used...

Reply:
Your call!